File: CSharp/06. Product Activation/01. Activation/sample.cs
using SautinSoft;
using System.IO;
using static SautinSoft.HtmlToRtf;

namespace Sample
{
    class Sample
    {
        static void Main()
        {
			// Get your free key here:   
            // https://sautinsoft.com/start-for-free/
	
            // You will get own serial number after purchasing the license.
            // If you will have any questions, email us to sales@sautinsoft.com or ask at online chat https://www.sautinsoft.com.
            // Let us say, you have this key: 1234567890.            

            // Activation of Html to Rtf .Net after purchasing.
            HtmlToRtf.SetLicense("1234567890");

            HtmlToRtf h = new HtmlToRtf();
            string inpFile = @"..\..\..\Sample.html";
            string outFile = @"Result.rtf";

            HtmlConvertOptions opt = new HtmlConvertOptions();
            opt.OutputFormat = HtmlToRtf.OutputFormat.Rtf;

            if (h.Convert(inpFile, outFile, opt))
            {
                // Open the result for demonstration purposes.
                System.Diagnostics.Process.Start(new System.Diagnostics.ProcessStartInfo(outFile) { UseShellExecute = true });
            }
        }
    }
}

File: VB.Net/02. HTML to RTF/01. Convert HTML to RTF file/sample.vb
Imports System
Imports System.IO
Imports SautinSoft
Imports SautinSoft.HtmlToRtf

Namespace Sample
	Friend Class Test

		Shared Sub Main(ByVal args() As String)
			' Get your free key here:   
            ' https://sautinsoft.com/start-for-free/
	
			' Convert HTML file to RTF file.
			' If you need more information about "HTML to RTF .Net" 
			' Email us at: support@sautinsoft.com.
			ConvertHtmlToRtfFile()
		End Sub

		Public Shared Sub ConvertHtmlToRtfFile()
			Dim h As New SautinSoft.HtmlToRtf()
			Dim opt As New HtmlConvertOptions()
			opt.OutputFormat = HtmlToRtf.OutputFormat.Rtf

			Dim inpFile As String = "..\..\..\Sample.html"
			Dim outFile As String = "Result.rtf"

			If h.Convert(inpFile, outFile, opt) Then
				' Open the result for demonstration purposes.
				System.Diagnostics.Process.Start(New System.Diagnostics.ProcessStartInfo(outFile) With {.UseShellExecute = True})
			End If
		End Sub
	End Class
End Namespace
